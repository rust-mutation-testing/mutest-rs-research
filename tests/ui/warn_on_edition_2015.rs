//@ print-mutants
//@ build
//@ stdout
//@ stderr
//@ edition: 2015

fn mutated_fn(a: i32, b: i32) -> i32 {
    a + b
}

#[test]
fn test() {
    assert_eq!(11, mutated_fn(5, 6));
}
