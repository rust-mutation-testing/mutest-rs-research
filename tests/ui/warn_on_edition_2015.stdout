1 mutation
  - [arg_default_shadow] ignore `a` argument by shadowing it with `Default::default()` in mutated_fn at tests/ui/warn_on_edition_2015.rs:7:15: 7:21
    <-(0)- test

1 mutation
  - [arg_default_shadow] ignore `b` argument by shadowing it with `Default::default()` in mutated_fn at tests/ui/warn_on_edition_2015.rs:7:23: 7:29
    <-(0)- test

1 mutation
  - [math_op_add_sub_swap] swap operator `+` for `-` in mutated_fn at tests/ui/warn_on_edition_2015.rs:8:5: 8:10
    <-(0)- test

1 mutation
  - [math_op_add_mul_swap] swap operator `+` for `*` in mutated_fn at tests/ui/warn_on_edition_2015.rs:8:5: 8:10
    <-(0)- test

4 mutants; 4 mutations; 4 safe; 0 unsafe (0 tainted); 0 batched; 4 unbatched
